src/rulesets/LinearAlgebra/dense.jl

```diff
@@ -394,3 +394,83 @@ function rrule(
     end
     return Ω, lyap_pullback
 end
+
+#####
+##### `kron`
+#####
+
+function frule((_, Δx, Δy), ::typeof(kron), x, y)
+    return kron(x, y), kron(Δx, y) + kron(x, Δy)
+end
+
+function rrule(::typeof(kron), x::AbstractVector, y::AbstractVector)
+    function kron_pullback(z̄)
+        x̄ = zero(x)
+        ȳ = zero(y)
+        m = firstindex(z̄)
+        @inbounds for i in eachindex(x)
```

Comment:
Is this safe (mainly thinking about indices of `z̄`)?

Reply:
It could well not be safe, I'm not sure.